src/goalServices/goals.js

```diff
@@ -2346,22 +2349,13 @@ export async function updateGoalStatusById(
     return false;
   }
 
-  // finally, if everything is golden, we update the goal
-  const g = await Goal.update({
-    status: newStatus,
-    closeSuspendReason,
-    closeSuspendContext,
-    previousStatus: oldStatus,
-  }, {
-    where: {
-      id: goalIds,
-    },
-    returning: true,
-    individualHooks: true,
-  });
-
-  const [, updated] = g;
-  return updated;
+  return Promise.all(goalIds.map((goalId) => changeGoalStatus({
```

Comment:
`changeGoalStatus` returns the goals that were updated, so `updateGoalStatusById` still has the same return value.